rs/orchestrator/src/orchestrator.rs

```diff
@@ -398,6 +402,28 @@ impl Orchestrator {
                                 break;
                             }
                         }
+
+                        let node_id = upgrade.node_id();
+                        match (&last_flow, &control_flow) {
+                            (
+                                OrchestratorControlFlow::Assigned(subnet_id),
+                                OrchestratorControlFlow::Leaving(_),
+                            ) => {
+                                UtilityCommand::notify_host(&format!("The node {node_id} has been unassigned from the subnet {subnet_id}\\
+                                     in the registry. Please do not turn off the machine while it completes its graceful removal from the subnet.\\
+                                      This process can take up to 15 minutes. A new message will be displayed here when the node has been \\
+                                      successfully removed."), 1);
```

Comment:
Did you try how this looks? I think you are escaping once too many:
```suggestion
                                UtilityCommand::notify_host(&format!("The node {node_id} has been unassigned from the subnet {subnet_id} \
                                     in the registry. Please do not turn off the machine while it completes its graceful removal from the subnet. \
                                      This process can take up to 15 minutes. A new message will be displayed here when the node has been \
                                      successfully removed."), 1);
```

Reply:
I didn't use a VM, probably should have logged it (I had a dummy log for that). You are right awesome catch!